shm_wrapper/shm_wrapper.c

```diff
@@ -311,7 +409,7 @@ void device_connect (uint16_t dev_type, uint8_t dev_year, uint64_t dev_uid, int
 		}
 	}
 	if (*dev_ix == MAX_DEVICES) {
-		printf("too many devices, connection unsuccessful\n"); //turn into a logger message later
+		log_runtime(WARN, "too many devices, connection unsuccessful");
```

Comment:
Change this to call error. Shouldn't be a warning I think

Reply:
Makes sense, will change~~

Reply:
Done